ui/bits/css/ublog/_card.scss

```diff
@@ -20,6 +20,12 @@
       0 0 20px $c-link;
   }
 
+  &--sticky {
```

Comment:
i know it's everywhere in our code, but using parent selector to glue names is bad for searchability, which means it is just bad.

Reply:
that's just an opinion, i'm not suggesting you rewrite all of our scss here! :smile:

Reply:
Yeap, I just copied the &--link above

I think I'll leave it like that for the time being. 
Is there a discussion somewhere about the best practices to adopt when working with scss ?